### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by morganestes** [AtomLinter/Linter#343]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/atom.js:55:11)
StatusBarSummaryView.render (/Users/morganestes/.atom/packages/linter/lib/statusbar-summary-view.coffee:11:21)
```

**Comment by morganestes** [comment]
I opened this via the one-click reporting tool inside Atom and let it populate the title and initial comment.

I just upgraded to 0.174.0, which introduced the Deprecation Cop feature to view and report anything that's calling the API differently than what is coming up with the 1.0 API (see http://blog.atom.io/2015/01/15/announcing-the-atom-1-api.html for details).

I don't know how critical it is to update, but I figured I'd go ahead and report it while I could.
